### PR TITLE
PLANNER-2862 Migrate examples to Jakson

### PR DESCRIPTION
**Pull request by dupliaka**
Added JacksonSolutionDao 
Updated datasets imported it from the examples
Included duplication functions

### CODE REVIEW

File: optaplanner-benchmarks/optaplanner-perf-benchmark/src/test/java/org/jboss/qa/brms/performance/SolverConfigFromXmlTest.java [hunk 1]
```diff
@@ -28,31 +28,30 @@
...
+                , Arguments.of(new ConferenceSchedulingBenchmark(ConferenceSchedulingExample.DataSet.TALKS_36_TIMESLOTS_12_ROOMS_5), Examples.CONFERENCE_SCHEDULING.getSolverConfigFromXml())
+                , Arguments.of(new FlightCrewSchedulingConstraintStreamsBenchmark(FlightCrewSchedulingExample.DataSet.EUROPE_175_FLIGHTS_7_DAYS), Examples.FLIGHT_CREW_SCHEDULING.getSolverConfigFromXml())
+                , Arguments.of(new NurseRosteringBenchmark(NurseRosteringExample.DataSet.SPRINT), Examples.NURSE_ROSTERING.getSolverConfigFromXml())
+                , Arguments.of(new TSPConstructionBenchmark(ConstructionHeuristicType.FIRST_FIT, TravelingSalesmanExample.DataSet.LU_980), Examples.TRAVELING_SALESMAN.getSolverConfigFromXml())
+                , Arguments.of(new TravelingTournamentConstraintStreamsBenchmark(TravelingTournamentExample.DataSet.SUPER_06), Examples.TRAVELING_TOURNAMENT.getSolverConfigFromXml())
+                , Arguments.of(new VehicleRoutingConstraintStreamsBenchmark(VehicleRoutingExample.DataSet.VRP_USA_100_10), Examples.VEHICLE_ROUTING.getSolverConfigFromXml())
```
Comment (rsynek) [inline optaplanner-benchmarks/optaplanner-perf-benchmark/src/test/java/org/jboss/qa/brms/performance/SolverConfigFromXmlTest.java:41]: If this formatting change intentional?

File: optaplanner-benchmarks/optaplanner-perf-framework/src/main/java/org/jboss/qa/brms/performance/examples/conferencescheduling/generator/ConferenceSchedulingGenerator.java [hunk 2]
```diff
@@ -213,12 +213,11 @@ public ConferenceSolution createConferenceSolution(int timeslotListSize, int roo
         labTalkCount = (dayListSize * labTimeslotCount) * labRoomCount;
 
         random = new Random(37);
-        ConferenceSolution solution = new ConferenceSolution();
-        solution.setId(0L);
+        ConferenceSolution solution = new ConferenceSolution(0l);
```
Comment (rsynek) [inline optaplanner-benchmarks/optaplanner-perf-framework/src/main/java/org/jboss/qa/brms/performance/examples/conferencescheduling/generator/ConferenceSchedulingGenerator.java:216]: `l` is often mistaken with `1`, hence it's better to use `L` for a `long` literal.
```suggestion
        ConferenceSolution solution = new ConferenceSolution(0L);
```

**Review by rsynek — APPROVED** [review]
I left a couple of formatting suggestions; otherwise looks good.

File: optaplanner-benchmarks/optaplanner-perf-benchmark/src/test/java/org/jboss/qa/brms/performance/SolverConfigFromXmlTest.java (hunk 1 above)
Comment (dupliaka) [reply optaplanner-benchmarks/optaplanner-perf-benchmark/src/test/java/org/jboss/qa/brms/performance/SolverConfigFromXmlTest.java:41]: Yes

**Pull request merged**